src/main/java/com/glodblock/github/common/parts/PartFluidPatternTerminalEx.java

```diff
@@ -90,7 +90,7 @@ public void readFromNBT( final NBTTagCompound data )
         this.crafting.readFromNBT( data, "craftingGrid" );
 
         this.setSubstitution( data.getBoolean( "substitute" ) );
-        this.setPrioritization( data.getBoolean( "priorization") );
+        this.setPrioritization( data.getBoolean( "prioritization") );
```

Comment:
you shouldn't change the nbt key value, otherwise it will lose data when updating from old version